libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/oracle/OBOracleSchemaAccessor.java

```diff
@@ -111,7 +113,7 @@ public DBDatabase getDatabase(String schemaName) {
     @Override
     public List<DBDatabase> listDatabases() {
         List<DBDatabase> databases = new ArrayList<>();
-        String sql = "SELECT USERNAME from ALL_USERS;";
+        String sql = "SELECT USERNAME from SYS.ALL_USERS;";
```

Comment:
why don't we use `dataDictTableNames.USERS()`?

Reply:
Because this is a String type sql, there is no need to use OracleSqlBuilder, so I just added the `SYS` prefix. It is better to use dataDictTableNames uniformly. I will modify it.